lib/redis_tls.rb

```diff
@@ -40,3 +40,38 @@ def parse_response(resp)
     arr[1]
   end
 end
+
+class SentinelTLS13
+  attr_reader :host, :port
+
+  def initialize(options)
+    @host = options.fetch(:host)
+    @port = options.fetch(:port)
+  end
+
+  def get_redis_instance
+    # no connection pooling for this example
+    sock = TCPSocket.new(@host, @port)
+    client = TTTLS13::Client.new(sock, @host)
+    client.connect
+
+    client.write("SENTINEL get-master-addr-by-name redis-master\n")
```

Comment:
(important) let's use master-name from the credentials we have instead of hard-coding it

Reply:
Done